common.gradle

```diff
@@ -34,8 +36,21 @@ android {
         manifestPlaceholders = [
                 appName : "Tachiyomi: $extName",
                 extClass: extClass,
-                nsfw: project.ext.find("isNsfw") ? 1 : 0,
+                nsfw    : project.ext.find("isNsfw") ? 1 : 0,
         ]
+        String baseUrl = project.ext.find("baseUrl") ?: ""
+        if (theme != null && !baseUrl.isEmpty()) {
```

Comment:
I think baseUrl field should be required if the theme lib have a deeplink manifest. Could add an extra boolean property to the themelib `requireBaseUrl` which governs this.

Reply:
It is guaranteed by AGP - if you do not fill in `baseUrl`, AGP will complain about missing substitution values.